cypress/component/DataSubmission/consent_group.spec.js

```diff
@@ -0,0 +1,59 @@
+import ConsentGroupForm from '../../../src/components/data_submitter/ConsentGroupForm';
+import { cloneDeep } from 'lodash/fp';
+import { mount } from 'cypress/react';
+
+const props = {
+  idx: 0,
+  parentConsentGroup: {},
+  saveConsentGroup: () => {},
+  deleteConsentGroup: () => {},
+}
+
+let propCopy = {}; 
+
+beforeEach(() => {
+    propCopy = cloneDeep(props);
+})
+
+describe('Consent Group', function () {
+  it('Edits without saving', function () {
+    cy.spy(propCopy, 'saveConsentGroup');
+
+    mount(<ConsentGroupForm {...propCopy}/>);
+
+    cy.get('#0_consent_group_name').type('Hello!');
+    cy.get('#0_url').type('https://www.asdf.gov');
+
+    expect(propCopy.saveConsentGroup).to.not.be.called;
+
+  }),
+  it('Saves properly', function () {
+    cy.spy(propCopy, 'saveConsentGroup');
+
+    mount(<ConsentGroupForm {...propCopy}/>);
+
+    cy.get('#0_consent_group_name').type('Hello!');
+    cy.get('#0_url').type('https://www.asdf.gov');
+    cy.get('#0_hmb').check();
+    cy.get('#0_col').check();
+    cy.get('#0_saveConsentGroup').click().then(() => {
+        expect(propCopy.saveConsentGroup).to.be.calledWith({
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 6 spaces but found 8.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9969451)
```suggestion
      expect(propCopy.saveConsentGroup).to.be.calledWith({
```